packages/dataviews/src/components/index.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import { DataViews as DataViewsRoot } from './dataviews';
+import DataViewsSearch from './dataviews-search';
+
+type DataViewsProps = typeof DataViewsRoot & {
+	Search: typeof DataViewsSearch;
+};
+
+const DataViews = DataViewsRoot as DataViewsProps;
+
+// Expose DataViews components sub components
+DataViews.Search = DataViewsSearch;
```

Comment:
Just noting that this pattern is going to make these components non tree-shakeable, from my perspective, that's fine since most dataviews are going to use most of the components. 

The alternative would be to export `DataViewsSearch` component on its own... but I personally prefer the current approach used in the PR as these components are highly coupled.

Reply:
Sounds reasonable to me. Going to work on a follow-up to expose the dataviews sub components . Cleaning.